<ui version="4.0" >
 <class>OperatorsDialog</class>
 <widget class="QDialog" name="OperatorsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Operators:</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/resources.qrc" >:/32/user.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Find:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findlineEdit" />
       </item>
       <item>
        <widget class="QToolButton" name="toolButton" >
         <property name="text" >
          <string>x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addpushButton" >
         <property name="text" >
          <string>Add</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelpushButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QTableView" name="tableView" >
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findlineEdit</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>addpushButton</tabstop>
  <tabstop>cancelpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>findlineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>165</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel" >
     <x>119</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>OperatorsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>194</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel" >
     <x>247</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
